docs/xks/developer-guide/scheduling.md

```diff
@@ -0,0 +1,138 @@
+---
+id: scheduling
+title: Scheduling Pods
+---
+
+## Scheduling
+
+### Kubernetes Scheduler
+
+The kube-scheduler looks for newly created pods that doesnt have any assigned node. The nodes that meet the requirements to run a pod is called a feasible node.
+
+ Kube-scheduler becomes responsible for assigning a suitable node to the pod based on 2 criteria:
+
+1. Filtering - Checks available nodes for available resources, to meet requirements specificed in the pod.
+2. Scoring - Based on the available resources when Filtering, ranking the avaliable nodes and choosing the highest ranking one, if 2 or more are equal, selection is random.
+
+### Node Selector
+
+You can schedule pods based on labels and nodeSelector, for example: `gpuEnabled: true`
+
+Nodeconfig:
+
+```yaml
+lables:
+    gpuEnabled: true
+```
+
+Podconfig:
+
+```yaml
+nodeSelector:
+    gpuEnabled: true
+```
+
+## Affinity & antiAffinity
+
+This is similar to `nodeSelector` bur greatly enhances the types of constraints you can express.
+
+1. The affinity/anti-affinity language is more expressive. The language offers more matching rules.
+2. Rules can be "preferred" rather than a hard requirement, so if the scheduler can't satisfy it, the pod will still be scheduled

```

Comment:
```suggestion
2. Rules can be "preferred" rather than hard requirements, so if the scheduler can't satisfy them, the pod will still be scheduled

```